notebooks/ICESat-2_MODIS_Arctic_Sea_Ice/Customize and Access Data.ipynb

```diff
@@ -218,11 +222,10 @@
     "    # Create session to store cookie and pass credentials to capabilities url\n",
     "    session = requests.session() \n",
     "    s = session.get(capability_url)\n",
-    "    response = session.get(s.url,auth=(uid,pswd))\n",
+    "    response = session.get(s.url, headers={\"Authorization\": f\"Bearer {auth}\"})\n",
     "    response.raise_for_status() # Raise bad request to check that Earthdata Login credentials were accepted \n",
     "    \n",
     "    #This function provides a list of all available services\n",
-    "    #orignal code: fn.print_service_options(data_dict, response)\n",
     "    fn.print_service_options(list(data_dict.values())[i], response) # not sure if the syntax is correct for a loop"
```

Comment:
I don't recall who wrote whether this is correct syntax or not- we should confirm this! The code works but I'm not sure if it's correct either.